runtime/reconcilers/model.go

```diff
@@ -196,9 +218,21 @@ func (r *ModelReconciler) Reconcile(ctx context.Context, n *runtimev1.ResourceNa
 
 	// Reschedule if we're not triggering
 	if !trigger {
+		// Show if any splits errored
+		if model.State.SplitsHaveErrors {
```

Comment:
Yeah I was a little torn on this. My reasoning was that the current logic is designed to *always* trigger if a previous invocation returned an error (by only changing the `trigger` and `refreshed_on` fields after a successful execution). But with the splits that is no longer the case (it will not auto-retrigger if a split errored), so that's why I added this to make it clear that there are errors to attend to.